Add I18N Support

2ykwang opened the pull request:
related issue:  #542

**support 22 languages (using LLM)**
Arabic, Bulgarian, Catalan, Czech, German, Spanish, Argentine Spanish, 
Persian, Finnish, French, Hebrew, Italian, Japanese, Kazakh, Korean, 
Dutch, Polish, Portuguese (Brazil), Russian, Slovak, Turkish, Chinese (Simplified)


- Added `.mo` files (removed from `.gitignore`)
- Updated hardcoded strings to use gettext (`_("")`)
- Added `justfile` to support i18n update workflow

hramezani commented:
Thanks @2ykwang for the PR. 

How did you translate the keywords? did you use any AI?

2ykwang commented:
@hramezani yes, I translated Korean and Japanese manually and used an LLM for the others. 
the keywords are simple, so I see this as a base for the community to refine.

2ykwang commented:
@hramezani I’ve kept only Japanese and Korean for now, and removed the other languages to encourage community contributions.
also added a just command to generate the base .po files.

hramezani commented on justfile:
```diff
@@ -0,0 +1,38 @@
+# Default recipe that lists all available commands
```
Why didn't you use `Makefile`? Do we need to install `just`?

2ykwang replied:
@hramezani `Justfile` isn’t as much of a “standard” as `Makefile`, but it’s now fairly common in open-source projects as a task runner. 
for `django-auditlog` porject needs (i18n, test, install), just’s command-runner model felt like a better fit. 
Its simpler syntax also makes commands easier to maintain, and it works consistently across Windows, macOS, and Linux.

2ykwang replied:
>  also we need to install just to use it. I prefer Makefile because it is common and can cover our use case.

@hramezani makes sense. I picked Justfile since I had seen and used it frequently in past projects, but I agree Makefile is the right fit here since it’s dependency-free and already the Jazzband convention.

hramezani commented:
Thanks @2ykwang
